### ISSUE TITLE
`autoResize` undocumented?

### ISSUE THREAD

**Issue by t3db0t** [JackAdams/meteor-editable-text#76]
I've been trying for a few hours now to hack together an auto-resizing editable-text input field, and I stumbled upon the undocumented `autoResize` option.  It seems to work, if slightly laggy; is it unfinished?

**Comment by t3db0t** [comment]
Ah, I see now the lag issue while typing, and the +20px offset to try to make it less obvious.  Hmmmm.

**Comment by JackAdams** [comment]
Yeah, it's undocumented because it's not very well implemented.  There are a few other undocumented bits and pieces too.

**Comment by t3db0t** [comment]
There's got to be a way!

**Comment by JackAdams** [comment]
I'll accept a PR if you get a workable solution going. The relevant lines of code are: https://github.com/JackAdams/meteor-editable-text/blob/master/lib/editable_text.js#L394-L403
